migrations/2018-04-17-subscribers-table.sql

```diff
@@ -0,0 +1,12 @@
+-- This file creates a new table, subscriptions, in the ejabberd DB.
+
+USE ejabberd;
+
+CREATE TABLE subscriptions (
```

Comment:
Nit, but our standard is not to pluralize table names in TS and I see table names like `archive` here, so I'd drop the `s`

Reply:
But if it's easier to leave this this way for existing code, then just ignore me